### ISSUE TITLE
train with custom dataset

### ISSUE THREAD

**Issue by liuxiaoyu1104** [robincourant/DIRECTOR#3]
If I have custom  camera poses (which may be labeled differently from those in your dataset), what should I consider if I want to train using your code? Specifically, should I recalculate the mean and variance for the data, and is it necessary to adjust the variance of the noise added during training and sampling?

**Comment by robincourant** [comment]
Hi,

If you plan to train the model on a different dataset, I highly recommend recomputing the mean and variance and storing them [there](https://github.com/robincourant/DIRECTOR/blob/main/configs/dataset/standardization/0300.yaml) - this should ensure things work correctly. 
As for the diffusion hyperparameters, I wouldn't touch them, as the EDM framework handles that well. 
However, you might need to fine-tune some [training hyperparameters](https://github.com/robincourant/DIRECTOR/blob/main/configs/diffuser/optimizer/adamW.yaml), such as the learning rate (lr).

Keep me in touch if you face any troubles!

**Comment by liuxiaoyu1104** [comment]
Thank you so much for your reply. I would like to ask, do you have any suggestions for tuning some training hyperparameters? My dataset consists of approximately 3K videos.

**Comment by robincourant** [comment]
Could you give me more details to help you with this?
Have you already tried launching a training? Do you have the loss curves? What are the current results?
